AppCenter/AppCenterTests/MSAuthTokenContextTests.m

```diff
@@ -34,15 +34,44 @@ - (void)testSetAuthToken {
 
   // If
   NSString *expectedAuthToken = @"authToken1";
+  NSString *expectedAccountId = @"account1";
+  NSString *expectedAccountId2 = @"account2";
   id<MSAuthTokenContextDelegate> delegateMock = OCMProtocolMock(@protocol(MSAuthTokenContextDelegate));
   [self.sut addDelegate:delegateMock];
 
   // When
-  self.sut.authToken = expectedAuthToken;
+  [self.sut setAuthToken:expectedAuthToken withAccountId:expectedAccountId];
 
   // Then
-  XCTAssertEqualObjects(self.sut.authToken, expectedAuthToken);
+  XCTAssertEqualObjects([self.sut authToken], expectedAuthToken);
+  OCMVerify([delegateMock authTokenContext:self.sut didUpdateUserWithAuthToken:expectedAuthToken]);
+  
+  // When
+  [self.sut setAuthToken:expectedAuthToken withAccountId:expectedAccountId];
+  
+  //Then
```

Comment:
Could you please run ./clang-format-changed-files.sh utility to fix formatting.

Reply:
It looks like you reviewed not updated version. It's already fixed in https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1328/commits/9d43dc302e791419bbf128ed9597e6759e08b7e5